yarn-project/stdlib/src/tx/tx.ts

```diff
@@ -170,6 +171,40 @@ export class Tx extends Gossipable {
     return logsSource.getPublicLogs({ txHash: await this.getTxHash() });
   }
 
+  /**
+   * Gets either revertible or non revertible contract class logs emitted by this tx.
+   * @param revertible - true for revertible only logs, false for non reverible only logs.
```

Comment:
Oh gosh, I didn't even pay attention to any logic that discerned between rev and non-rev in the `.nr` files...

Reply:
I didn't need to change it because before this PR we use a log hash in the kernels. After this PR we... use a log hash in the kernels :)